apps/codeimage/src/components/Pagination/buttons.ts

```diff
@@ -1,5 +1,17 @@
 import {arrayFomRange} from '@core/modules/pagination/arrayFromRange';
 import {Accessor, mapArray, Setter} from 'solid-js';
+
+/**
+ *  This hook allow us to create a list of props to render buttons of our pagination component
+ * @param lastPage callback that return number of the lastPage of paginations Button
+ * @param selectedPage callback that return number of the pageSelected
+ * @param onChange callback that allow change page
+ * @param disabled boolean that
+ * @returns the list of the props buttons
+ *
+ *
+ * @examples
+ */
```

Comment:
this is only as a placeholder to try doc our hooks, WDYT @riccardoperra ? Maybe it's better to do in type of the function?

Reply:
as a TS project JSDOC param property should not be used. Instead, handling the comment inside each property is recommended, since the types are already defined

```ts
export const createPaginationButtons = (
  /**
   * Accessor that return number of the lastPage of pagination Button
   */
  lastPage: Accessor<number>,
  /**
   * Accessor that return number of the pageSelected
   */
  selectedPage: Accessor<number>,
) => {};
```

Note that I'm also using the **Accessor** prefix instead of callback since in solid even though they are both functions, in solid the accessors have a more precise purpose